src/redux/selectors/index.js

```diff
@@ -211,7 +213,11 @@ const filteredItemsInDateRangeSelector = createSelector(
 );
 
 const sortedGroupedRecordsByType = (records, isDescending) => {
-  const sortedRecords = isDescending ? [...records].reverse() : records;
+  // eslint-disable-next-line no-nested-ternary
+  const sortedRecords = isDescending === undefined
```

Comment:
`isDescending` can be undefined?

Reply:
`sortedGroupedRecordsByType` is used by the `SubTypeAccordionContainer` in `CatalogScreen` and in multiple places inside `DetailsPanel`

The `SubTypeAccordionContainer` in `CatalogScreen` cannot be resorted (should always be descending). So this instance of `SubTypeAccordionContainer` does not get a `isDescending` prop.

To make this more obvious, I could make a prop called `fromCatalogScreen` which would negate any reordering of records.